sky/setup_files/setup.py

```diff
@@ -87,7 +101,15 @@
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Topic :: System :: Distributed Computing',
     ],
-    description='SkyPilot',
-    long_description=__doc__.replace('\n', ' '),
+    description='SkyPilot: An intercloud broker above the clouds',
```

Comment:
above the clouds -> for the clouds?

This description probably can use iterations (can do later).

Reply:
Fixed. Thanks!